src/main/java/com/exactpro/th2/common/schema/cradle/CradleConfiguration.java

```diff
@@ -43,4 +42,88 @@
 
     @JsonProperty
     private String cradleInstanceName;
+
+    /**
+     * This is the max of the event batch size in bytes with {@link CassandraStorageSettings#DEFAULT_MAX_EVENT_BATCH_SIZE} as the default value.
+     * The option is used as an argument of the {@link CradleManager#initStart(String, boolean, long, long)} method
+     */
+    private long cradleMaxEventBatchSize = CassandraStorageSettings.DEFAULT_MAX_EVENT_BATCH_SIZE;
+
+    /**
+     * This is the max of the message batch size in bytes with {@link CassandraStorageSettings#DEFAULT_MAX_MESSAGE_BATCH_SIZE} as the default value.
+     * The option is used as an argument of the {@link CradleManager#initStart(String, boolean, long, long)} method
+     */
+    private long cradleMaxMessageBatchSize = CassandraStorageSettings.DEFAULT_MAX_MESSAGE_BATCH_SIZE;
```

Comment:
We should also add javadoc for getters/setters of these fields

Reply:
done